### DISCUSSION TITLE
DAI 2024-2025 - Practical work 3 - Tristan Baud, Mario Ferreira and Mathieu Emery

### DISCUSSION THREAD

**Discussion by mathieuemery** [heig-vd-dai-course/heig-vd-dai-course#544] [Show and tell]
This project has the goal to develop a dynamic interface that allows users to manage OKRs (Objectives and Key Results).

**Comment by ludelafo** [comment]
Looks good to us, but please be aware that not everyone knows what OKRs are (we had to search on Wikipedia :)).

**Comment by mathieuemery** [comment]
Hello,

Here is the link to our GitHub repository: https://github.com/HEIG-VD-S3-PW/DAI-LAB-04

And here is the last commit hash: f806560

Best regards

**Reply by ludelafo** [reply]
Hello @mathieuemery,

I am not able to access your project on <https://api-heig-dai-pw04.duckdns.org>. Is the server running?

Thank you in advance,
Ludovic

**Comment by mathieuemery** [comment]
Hello @ludelafo,

My colleague is starting it right now, sorry for the inconvenience.

**Reply by ludelafo** [reply]
Thank you!
